lib/palindrome_permutation.rb

```diff
@@ -1,4 +1,25 @@
 
 def palindrome_permutation?(string)
+  letter_count = {}
+  odd_counts = 0
   
+  string.each_char do |char|
+    if letter_count[char]
+      letter_count[char] += 1
+    else
+      letter_count[char] = 1
+    end
+  end
+  
+  letter_count.each do |count, value|
+    if value % 2 == 1
```

Comment:
Ruby does actually have a `.odd?` method.  Not really a problem, just something to note.